extension/tests/integration_tests.rs

```diff
@@ -50,6 +50,20 @@ async fn test_scheduled_job() {
     .expect("failed to select from test_table");
     // 3 rows returned
     assert_eq!(result.rows_affected(), 3);
+
+    let result_hybrid = sqlx::query(&format!(
+        "SELECT vectorize.hybrid_search(
+        job_name => '{job_name}',
+        query => 'mobile devices',
+        return_columns => ARRAY['product_name'],
+        num_results => 3
+    );"
+    ))
+    .execute(&conn)
+    .await
+    .expect("failed to select from test_table");
+    // 3 rows returned
+    assert_eq!(result_hybrid.rows_affected(), 3);
```

Comment:
I don't quite see how this fits into `test_scheduled_job()`. Maybe create a new test?